### PR TITLE
[sflow][db_migrator] Egress Sflow support

### PR DESCRIPTION
**Pull request by rajkumar38**
#### What I did
Based on review comments from PR https://github.com/sonic-net/sonic-utilities/pull/2790, raised a separate PR for db_migrator changes as part of egress sFlow feature.
#### How to verify it
Recompile sonic-utilities bdist_wheel and verify pytest

### CODE REVIEW

**Comment by rajkumar38** [comment]
@prsunny @vaibhavhd @qiluo-msft 
As suggested, raised new PR for db_migrator changes.
Pls review and approve.

**Comment by zhangyanzhao** [comment]
@qiluo-msft  @madhupalu  @vadymhlushko-mlnx  can you please help to review this PR? Thanks.

File: scripts/db_migrator.py [hunk 1]
```diff
@@ -1056,9 +1072,31 @@ def version_4_0_3(self):
     def version_4_0_4(self):
         """
         Version 4_0_4.
-        This is the latest version for master branch
         """
         log.log_info('Handling version_4_0_4')
+
+        sflow_tbl = self.configDB.get_table('SFLOW')
+        for k, v in sflow_tbl.items():
```
Comment (prsunny) [inline scripts/db_migrator.py:1079]: Please move this to migrate_sflow function.

Comment (rajkumar38) [reply scripts/db_migrator.py]: Fixed. Pls review

**Review by prsunny — APPROVED** [review]

**Pull request merged**